### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by jpnurmi**

### CODE REVIEW

**Comment by jpnurmi** [comment]
Screenshot from 2022-03-29 13-04-43
Screenshot from 2022-03-29 13-05-03
Screenshot from 2022-03-29 13-05-18

**Comment by jpnurmi** [comment]
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

**Comment by jpnurmi** [comment]
# New issue
image

# Bug report
image

# Feature request
image

**Review by seb128 — CHANGES_REQUESTED** [review]
Thanks for working on this and sorry for not picking it up earlier. It seems fine to me, I'm unsure if we should start by asking the version of Ubuntu/the ISO but that probably doesn't matter as long as we are rolling and we don't have different series available.

I've one small tweak request and then let's merge.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      1. Go to '...'
+      2. Click on '...'
+      3. Scroll down to '...'
+      4. See '...'
+- type: textarea
+  attributes:
+    label: Additional context
+    description: |
+      Logs, links, screenshots, videos, or anything that may help us to resolve the issue.
+    placeholder: |
```
[the remark this answers is not in the document]
Comment (jpnurmi) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:28]: What about proposing a command that collects all relevant logs?

**Comment by seb128** [comment]
I think we could land the current version and iterate later if there are improvements we want to do

**Pull request merged**